torch/onnx/verification.py

```diff
@@ -136,136 +73,271 @@ def _run_ort(ort_sess, inputs):
     for input_name, input in kw_inputs.items():
         ort_inputs[input_name] = _to_numpy(input)
     inputs = _to_numpy(inputs)
-    ort_sess_inputs = ort_sess.get_inputs()
+    ort_session_inputs = ort_session.get_inputs()
     for i, input in enumerate(inputs):
-        if i == len(ort_sess_inputs) or ort_sess_inputs[i].name in ort_inputs:
+        if i == len(ort_session_inputs) or ort_session_inputs[i].name in ort_inputs:
             raise ValueError(
                 f"got too many positional inputs. inputs: {inputs}. kw_inputs: {kw_inputs}"
             )
-        ort_inputs[ort_sess_inputs[i].name] = input
-    ort_outs = ort_sess.run(None, ort_inputs)
+        ort_inputs[ort_session_inputs[i].name] = input
+    ort_outs = ort_session.run(None, ort_inputs)
     return _inline_flatten_list(ort_outs, [])
 
 
-def _ort_compare_with_pytorch(ort_outs, output, rtol, atol):
-    output, _ = torch.jit._flatten(output)
-    outputs = _unpack_to_numpy(output)
+def _create_ort_session(
```

Comment:
Omitting prefixes like "create", "get", "new", etc in the function name is often better if a function name is still clear in context without it. I believe here it would be clear, so I suggest:
```suggestion
def _ort_session(
```